src/main/scala/cosas/ops/types/SerializeDenotations.scala

```diff
@@ -0,0 +1,44 @@
+package ohnosequences.cosas.ops.types
+
+import ohnosequences.cosas._, types._, typeSets._, fns._
+
+// TODO msg not found
+trait SerializeDenotations[Denotations <: AnyTypeSet, V]
+extends Fn2[Denotations, Map[String,V]] with
```

Comment:
I don't understand what you are talking about. If I have a typeset/record and I want to serialize it to a Map, I see very logical to get a map with elements of this typeset/record (not any other things). Now, if I have two typesets and I want to get a union or check that they don't have common elements — whatever, I will do it on the typesets. If I already serialized them, I'm working with Maps, no more compile time guarantees or whatever, it's just Maps.